core/src/main/java/greencity/controller/AdviceController.java

```diff
@@ -1,8 +1,7 @@
 package greencity.controller;
 
-import static greencity.constant.ErrorMessage.INVALID_HABIT_ID;
-
-import greencity.constant.AppConstant;
+import greencity.annotations.ApiLocale;
+import greencity.annotations.ValidLanguage;
```

Comment:
Does `LanguageValidator `should work now with generic type `Locale `instead of `String`?

Reply:
Yea. I changed it, so it works fine

Reply:
Agree, I didn't notice.